handbook/people-ops/onboarding/index.md

```diff
@@ -18,6 +18,50 @@ At Sourcegraph, we don't think of teammates as resources — we think of them as
 
 Never assume that a problem is somebody else's to fix!
 
+## Onboarding standards and success
```

Comment:
E.g., otherwise I'm afraid the headers/sections below would be confused with the content on https://about.sourcegraph.com/handbook/people-ops/onboarding. A dedicated page might make it more clear that this is just about _expectations_.

Reply:
@dadlerj Yes, I'm totally okay with that! Let's make it its own page 👍

Reply:
After starting to do this, I'm changing my mind 😅 ... The more I try to move it out, the more I think that leaving it in this file it best. I'll approve, no need to split this out!